include/vinecopulib/bicop/implementation/class.ipp

```diff
@@ -54,7 +54,15 @@ inline Bicop::Bicop(const boost::property_tree::ptree input)
           input.get<int>("rotation"),
           tools_serialization::ptree_to_matrix<double>(
             input.get_child("parameters")))
-{}
+{
+  var_types_ = tools_serialization::ptree_to_vector<std::string>(
+    input.get_child("var_types"));
+  // try block for backwards compatibility
```

Comment:
If you really want backwards compatibility, shouldn't you also include the previous 2 lines in the try block?